Source/LinqToDB.EntityFrameworkCore/ILinqToDBForEFTools.cs

```diff
@@ -120,5 +122,9 @@ public interface ILinqToDBForEFTools
 		/// </summary>
 		bool EnableChangeTracker { get; set; }
 		
+		/// <summary>
+		/// List of interceptors that should be added by default to data connections/contexts created from EF Core provider
+		/// </summary>
+		List<IInterceptor> DefaultLinq2DbInterceptors { get; set; }

```

Comment:
A `List<>` should not be part of the public API. Also, there is no reason for it to be settable - you can initialize to a `List<>` internally and know that no one has set it to `null`. 

```suggestion
		IList<IInterceptor> DefaultLinq2DbInterceptors { get; }
```

Reply:
You are right - I was thinking about this already to be honest. Adjusted.